packages/components/src/components/expandable/expandable.test.ts

```diff
@@ -99,11 +99,20 @@ describe('<sd-expandable>', () => {
       </sd-expandable>
     `);
     const toggleButton = el.shadowRoot?.querySelector('button');
+    const detailsElement = el.shadowRoot?.querySelector('[part="details"]');
+
     toggleButton?.click(); // Simulate user clicking the toggle button
 
+    await el.updateComplete;
+
     expect(el.open).to.be.true;
+    expect(detailsElement?.hasAttribute('inert')).to.be.false;
```

Comment:
Instead of testing the attribute, shouldn't we check instead if things are focusable when closed?

Reply:
Added new test